guide/src/client/repositories.md

```diff
@@ -0,0 +1,68 @@
+# Northstar Repositories
+
+Repositories are used to group sets of containers. These are specified in
+the Northstar configuration (see `northstar.toml`).
+
+Containers can be added or removed from repositories at runtime by
+**installing** and **uninstalling** them. Note that repositories will not allow
+these actions if they are configured with the `writable` parameter set to
+`false`.
```

Comment:
I think we do not have this option.

Reply:
You are right, funny that this doc is already outdated :')